Header header
sensor_msgs/PointCloud2 obstacle_pointcloud
sensor_msgs/PointCloud2 target_pointcloud
std_msgs/Float32MultiArray grasp_poses
float64[] start_joint
float64[] scores
